config.toml.sample

```diff
@@ -1,54 +1,55 @@
+signupSupport = false                   # Enable/disable signup feature support. Manager plugin is required.
+allowSignout = false                    # Let users signout from service. Signup plugin is required.
+allowAnonymousChangePassword = false    # Enable/disable anonymous user can send change password email. Manager plugin is required.
+allowProjectResourceMonitor = true      # Allow users to look up its group monitor statistics
+allowManualImageNameForSession = false  # If true, user will be able to use the specific environment image by typing the exact name.
+allowSignupWithoutConfirmation = false  # If true, token received from confirmation message is ignored.
+autoLogout = false                      # If true, user will be automatically logout when they close all Backend.AI tab / window.
+#loginAttemptLimit = 5                  # The number of allowed login attempt for preventing from brute-force attacks
+#loginBlockTime = 180                   # The amount of time(sec.) for rejecting login attempt for preventing from brute-force attacks
+debug = false                           # Please refer to `[Debug mode](https://github.com/lablup/backend.ai-webui#debug-mode)` in README
+maskUserInfo = false                    # If true, then account information in header, session list will be masked
 
 [wsproxy]
-proxyURL = "http://127.0.0.1:5050/"
-#proxyURL = "https://wsproxy.onprem.backend.ai:9443/"
-#proxyBaseURL = ""
-#proxyListenIP = ""
-autoPortSelection = true
-disableCertCheck = true
+proxyURL = "[Proxy URL]"
+proxyBaseURL = "[Base URL of websocket proxy,]"
+proxyListenIP = "[Websocket proxy configuration IP.]"
+#disableCertCheck = false                # If true, It will disable certification check
 
 [menu]
-#blocklist = "summary,import,statistics" # Separated by comma.
+#blocklist = "summary,import,statistics" # Disable menus on the list which are separated by comma.
 
 [resources]
-openPortToPublic = false # Show option to open app proxy port to anyone.
-maxCPUCoresPerContainer = 256 # Maximum CPU per container.
-maxMemoryPerContainer = 64 # Maximum memory per container.
-maxCUDADevicesPerContainer = 8  # Maximum CUDA devices per container.
-maxCUDASharesPerContainer = 8  # Maximum CUDA shares per container.
-maxShmPerContainer = 4 # Maximum shared memory per container.
-maxFileUploadSize = 4294967296 # Maximum size of single file upload. Set to -1 for unlimited upload.
+openPortToPublic = true          # Show option to open app proxy port to anyone.
+maxCPUCoresPerContainer = 256    # Maximum CPU per container.
+maxMemoryPerContainer = 64       # Maximum memory per container.
+maxCUDADevicesPerContainer = 16  # Maximum CUDA devices per container.
+maxCUDASharesPerContainer = 8    # Maximum CUDA shares per container.
+maxShmPerContainer = 1           # Maximum shared memory per container.
+maxFileUploadSize = 4294967296   # Maximum size of single file upload. Set to -1 for unlimited upload.
```

Comment:
Is the changes in the default sample values for `openPortToPublic`, `maxShmPerContainer` okay?

Reply:
Yes. `openPortToPublic` should be `false` by default. I also think the sane default for `maxCUDA*PerContainer` would be `8`.

Reply:
Got it! I'll update those files right away.